test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -73,6 +73,36 @@ Describe "Get-Content" -Tags "CI" {
     It "should throw 'PSNotSupportedException' when you set-content to an unsupported provider" -Skip:($IsLinux -Or $IsMacOS) {
         {get-content -path HKLM:\\software\\microsoft -ea stop} | Should Throw "IContentCmdletProvider interface is not implemented"
     }
+    It 'Verifies -Tail reports a TailNotSupported error for unsupported providers' {
+        try {
+            Get-Content -Path Variable:\PSHOME -Tail 1 -ErrorAction Stop
+        }
+        catch {
+            $_.FullyQualifiedErrorId | Should Be 'TailNotSupported,Microsoft.PowerShell.Commands.GetContentCommand'
```

Comment:
Does this work in this instance?


```powershell
 { Get-Content -Path Variable:\PSHOME -Tail 1 -ErrorAction Stop }  | ShouldBeErrorId  'TailNotSupported,Microsoft.PowerShell.Commands.GetContentCommand'
```

Reply:
@markekraus: No, I could not get that pattern to work.

Reply:
fixed